source/operations/monitoring/collect-minio-metrics-using-prometheus.rst

```diff
@@ -15,7 +15,7 @@ Monitoring and Alerting using Prometheus
    - `Monitoring with MinIO and Prometheus: Overview <https://youtu.be/A3vCDaFWNNs?ref=docs>`__
    - `Monitoring with MinIO and Prometheus: Lab <https://youtu.be/Oix9iXndSUY?ref=docs>`__
 
-MinIO publishes cluster, node, and bucket metrics using the :prometheus-docs:`Prometheus Data Model <concepts/data_model/#data-model>`.
+MinIO publishes cluster, node, bucket, and resource metrics using the :prometheus-docs:`Prometheus Data Model <concepts/data_model/#data-model>`.
```

Comment:
https://github.com/minio/minio/releases/tag/RELEASE.2023-10-07T15-07-38Z

Oddly resource metrics were released in 10-07.

We should note that, maybe as a simple 

```
.. versionchanged:: RELEASE.2023-10-07T15-07-38Z

   `Resource metrics added`
```

Or osmething.

Reply:
Oh duh. Yes, version notice